plugins/s3-viewer-backend/src/service/standaloneServer.ts

```diff
@@ -34,22 +35,36 @@ export interface ServerOptions {
 export async function startStandaloneServer(
   options: ServerOptions,
 ): Promise<Server> {
-  const logger = options.logger.child({ service: 's3-viewer-backend' });
+  const logger = options.logger.child({ service: 's3-viewer' });
   const config = await loadBackendConfig({ logger, argv: process.argv });
-  const taskScheduler = TaskScheduler.fromConfig(config, {
-    logger: loggerToWinstonLogger(logger),
-  });
+  const taskScheduler = TaskScheduler.fromConfig(config, {});
   const scheduler = taskScheduler.forPlugin('s3-viewer');
   logger.debug('Starting application server...');
 
+  const discovery = HostDiscovery.fromConfig(config);
+  const tokenManager = ServerTokenManager.fromConfig(config, {
+    logger,
+  });
+
   const permissionRouter = await createPluginPermissions({ config, logger });
 
-  const router = await createRouter({ logger, config, scheduler });
+  const permissions = ServerPermissionClient.fromConfig(config, {
+    discovery,
+    tokenManager,
+  });
+
+  const router = await createRouter({
+    logger,
+    config,
+    scheduler,
+    discovery,
+    permissions,
+  });
 
   let service = createServiceBuilder(module)
     .setPort(options.port)
     .addRouter('/api/permission', permissionRouter)
-    .addRouter('/api/s3', router);
+    .addRouter('/api/s3-viewer', router);
```

Comment:
Is this url change is to be consistent with the typical user-facing route at `/s3-viewer`?

Reply:
Yes, this is basically because of the new backend-system name. It's just better to keep the naming consistent. Anyways, this is only relevant for the local testing

Reply:
Connected to this, I updated the docs as well: https://github.com/spreadshirt/backstage-plugin-s3/pull/90/commits/de0b7d8041106054433ab6e8171e62c3a7b5a505

Turns out this was already a problem because we forgot the update and it was reported in this issue: https://github.com/spreadshirt/backstage-plugin-s3/issues/92